rbs/SignatureTranslator.cc

```diff
@@ -10,44 +10,46 @@ namespace sorbet::rbs {
 
 namespace {
 
-rbs_string_t makeRBSString(const string_view &str) {
+rbs_string_t makeRBSString(const string &str) {
     return rbs_string_new(str.data(), str.data() + str.size());
 }
 
 } // namespace
 
 unique_ptr<parser::Node>
 SignatureTranslator::translateAssertionType(vector<std::pair<core::LocOffsets, core::NameRef>> typeParams,
-                                            const rbs::Comment &assertion) {
-    rbs_string_t rbsString = makeRBSString(assertion.string);
+                                            const rbs::RBSDeclaration &assertion) {
+    string signatureString = assertion.string();
```

Comment:
We need to hold the string value here to make sure it has the same lifecycle as this entire function.